dspace-api/src/main/java/org/dspace/content/DSpaceObject.java

```diff
@@ -119,12 +128,35 @@ protected void addDetails(String d) {
     }
 
     /**
+     * Deprecated: Use dedicated metadata event detail methods instead.
```

Comment:
Same here, we could be clearer by saying
`Deprecated: Use {@link #getMetadataEventDetails()} instead`